embedx/config.schema.json

```diff
@@ -1901,6 +1901,12 @@
               "Authorization": "Bearer some-token"
             }
           ]
+        },
+        "return_to_enabled": {
```

Comment:
not sure if this naming is the best here...

maybe I should make it `override_return_to`.